src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Properties/launchSettings.json

```diff
@@ -2,6 +2,7 @@
   "profiles": {
     "Microsoft.DotNet.UpgradeAssistant.Cli": {
       "commandName": "Project",
+      "commandLineArgs": "analyze -v -f html \"C:\\Users\\bleaphart\\Desktop\\BeanTrader.sln\"",
```

Comment:
we don't want to commit your local command line args. We keep this file so we can set the env variable to development

Reply:
Resolved